src/composables/useDocumentTitle.ts

```diff
@@ -0,0 +1,162 @@
+					&& newLastMessageMap[token] !== -1) // But is not from the current user
+		})
+		if (shouldShowAsterisk) {
+			document.title = '* ' + document.title
+		}
+	})
+
+	watch(isDocumentVisible, () => {
+		if (isDocumentVisible.value) {
+			// Remove asterisk for unread chat messages
+			if (document.title.startsWith('* ')) {
+				document.title = document.title.substring(2)
+			}
+		} else {
+			// Copy the last message map to the saved version,
+			// this will be our reference to check if any chat got a new
+			// message since the last visit
+			savedLastMessageMap.value = getLastMessageMap(conversationList.value)
+		}
+	})
+
+	/**
+	 * Adjust the page title to the conversation name once conversationsList is loaded
+	 */
+	EventBus.once('conversations-received', () => {
+		setPageTitleFromRoute(Router.currentRoute, false)
+	})
+
+	/**
+	 * Change the page title after the route was changed
+	 */
+	Router.afterEach((to) => setPageTitleFromRoute(to, false))
+
+	/**
+	 * Get a list for all last message ids
+	 *
+	 * @param conversationList array of conversations
+	 */
+	function getLastMessageMap(conversationList: Conversation[]): Record<string, number> {
+		if (conversationList.length === 0) {
+			return {}
+		}
+
+		return conversationList.reduce((acc: Record<string, number>, conversation: Conversation) => {
+			const { token, lastMessage } = conversation
+			// Default to 0 for messages without valid lastMessage
+			if (!lastMessage || Array.isArray(lastMessage)) {
+				acc[token] = 0
+				return acc
+			}
+
+			if (lastMessage.actorId === actorId.value && lastMessage.actorType === actorType.value) {
+				// Set a special value when the actor is the author so we can skip it.
+				// Can't use 0 though because hidden commands result in 0,
+				// and they would hide other previously posted new messages
+				acc[token] = -1
+			} else {
+				// @ts-expect-error: Property 'id' does not exist on type ChatProxyMessage
+				const lastMessageId = lastMessage.id ?? 0
+				const lastKnownMessageId = store.getters.getLastKnownMessageId(token) ?? 0
+				acc[token] = Math.max(lastMessageId, lastKnownMessageId)
+			}
+			return acc
+		}, {})
+	}
+
+	/**
+	 *
+	 * @param route current web route
+	 * @param showAsterisk Whether to show unread indicator
+	 */
+	function setPageTitleFromRoute(route: Route, showAsterisk: boolean) {
+		switch (route.name) {
+		case 'conversation':
+			setPageTitle(store.getters.conversation(route.params.token)?.displayName ?? '', showAsterisk)
+			break
+		case 'duplicatesession':
+			setPageTitle(t('spreed', 'Duplicate session'), showAsterisk)
+			break
+		default:
+			setPageTitle('', showAsterisk)
+		}
+	}
+
+	/**
+	 * Set the page title to the conversation name
+	 *
+	 * @param title Prefix for the page title e.g. conversation name
+	 * @param showAsterisk Whether to show unread indicator
+	 */
+	function setPageTitle(title: string, showAsterisk: boolean) {
```

Comment:
You are using this in `setPageTitleFromRoute` only which has `showAsterisk`  as `false` in both calls so it doesn't make sense to put this arg. Did you want to use it  in `conversationList` watcher maybe?

Reply:
Removed second argument, as asterisk adding/removing is handled by ref